commons/com.b2international.index/src/com/b2international/index/es/admin/EsIndexAdmin.java

```diff
@@ -224,17 +224,30 @@ public void create() {
 					SortedSet<String> incompatibleChanges = Sets.newTreeSet();
 					final JsonDiff schemaChanges = JsonDiff.diff(currentTypeMapping, newTypeMapping);
 					schemaChanges.forEach(change -> {
+						
 						if (change.isAdd()) {
+							
+							// XXX object type is the default type, so if the current mapping does not contain this node, we shouldn't trigger an update
+							if (change.getFieldPath().endsWith("type") && "object".equals(change.serializeValue())) {
```

Comment:
If this is always part of a path with more than one segment, the separator should be included to avoid partial word matches like eg. `classType`:

```suggestion
							if (change.getFieldPath().endsWith("/type") && "object".equals(change.serializeValue())) {
```

Reply:
👍🏻